packages/graphql/tests/integration/config-options/startup-validation.int.test.ts

```diff
@@ -20,15 +20,24 @@
 import type { Driver } from "neo4j-driver";
 import Neo4j from "../neo4j";
 import { Neo4jGraphQL } from "../../../src/classes";
+import { GraphQLError } from "graphql";
 
 describe("Startup Validation", () => {
     let driver: Driver;
     let neo4j: Neo4j;
 
-    const invalidTypeDefsError = 'Type "Point" already exists in the schema.';
+    const typePointAlreadyExistsErrors = [
+        new GraphQLError(
+            'Type "Point" already exists in the schema. It cannot also be defined in this type definition.'
+        ),
+        new GraphQLError(
+            'Field "Point.latitude" already exists in the schema. It cannot also be defined in this type extension.'
+        ),
+        new GraphQLError(
+            'Field "Point.longitude" already exists in the schema. It cannot also be defined in this type extension.'
+        ),
```

Comment:
Yeesh, it would be nice if we could clean this up, the 2nd and 3rd messages will just be confusing for users.

Reply:
Notice the "latitude" vs "longitude" difference. These errors were returned before this refactor. Should we just keep the first one?

Reply:
Oh they were already returned were they? It just feels a little confusing to me, because the latter two errors are kind of implied by the first error. But we can leave them if that's what the native GraphQL validation just does!